multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -24,20 +31,672 @@ use node_lib::database::db_initializer::{
+    // such an approach is implemented for another test in this file.)
+    let payment_thresholds = PaymentThresholds {
+        threshold_interval_sec: 2_500_000,
+        debt_threshold_gwei: 1_000_000_000,
+        payment_grace_period_sec: 85_000,
+        maturity_threshold_sec: 85_000,
+        permanent_debt_allowed_gwei: 10_000_000,
+        unban_below_gwei: 10_000_000,
+    };
+    let owed_to_serving_node_1_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) + 123_456;
+    let owed_to_serving_node_2_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) + 456_789;
+    let owed_to_serving_node_3_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) + 789_012;
+    let cons_node_initial_service_fee_balance_minor =
+        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei) * 4;
+    let exp_final_cons_node_service_fee_balance_minor = cons_node_initial_service_fee_balance_minor
+        - (owed_to_serving_node_1_minor
+            + owed_to_serving_node_2_minor
+            + owed_to_serving_node_3_minor);
+    let test_global_config = TestInputsOutputsConfig {
+        ui_ports_opt: None,
+        cons_node_initial_transaction_fee_balance_minor_opt: None,
+        cons_node_initial_service_fee_balance_minor,
+        debts_config: Either::Left(SimpleSimulatedDebts {
+            owed_to_serving_node_1_minor,
+            owed_to_serving_node_2_minor,
+            owed_to_serving_node_3_minor,
+        }),
+        payment_thresholds_all_nodes: payment_thresholds,
+        cons_node_transaction_fee_agreed_unit_price_opt: None,
+        exp_final_cons_node_transaction_fee_balance_minor: 999_842_470_000_000_000,
+        exp_final_cons_node_service_fee_balance_minor,
+        exp_final_service_fee_balance_serv_node_1_minor: owed_to_serving_node_1_minor,
+        exp_final_service_fee_balance_serv_node_2_minor: owed_to_serving_node_2_minor,
+        exp_final_service_fee_balance_serv_node_3_minor: owed_to_serving_node_3_minor,
+    };
+
+    let stimulate_payments =
+        |cluster: &mut MASQNodeCluster,
+         real_consuming_node: &MASQRealNode,
+         _global_test_config: &TestInputsOutputsConfig| {
+            for _ in 0..6 {
+                cluster.start_real_node(
+                    NodeStartupConfigBuilder::standard()
+                        .chain(Chain::Dev)
+                        .neighbor(real_consuming_node.node_reference())
+                        .build(),
+                );
+            }
+        };
+
+    let start_serving_nodes_and_run_check =
+        |cluster: &mut MASQNodeCluster,
+         serving_node_1_attributes: ServingNodeAttributes,
+         serving_node_2_attributes: ServingNodeAttributes,
+         serving_node_3_attributes: ServingNodeAttributes,
+         _test_global_config: &TestInputsOutputsConfig| {
+            let serving_node_1 = cluster.start_named_real_node(
```

Comment:
If you use this as the closure's parameter: `[ServingNodeAttributes; 3]`, then there would be less parameters in the closure.